examples/advanced/examples/encoding_sol_static.rs

```diff
@@ -0,0 +1,42 @@
+//! Example for encoding calldata via sol!
+
+use std::str::FromStr;
+
+use alloy::{
+    hex,
+    primitives::{Address, U256},
+    sol,
+    sol_types::SolCall,
+};
+
+sol!(
+    #[allow(missing_docs)]
+    function swapExactTokensForTokens(
```

Comment:
Perhaps an opportunity to link to https://docs.uniswap.org/contracts/v2/reference/smart-contracts/router-02#swapexacttokensfortokens so users know the origin of this specific function